src/libraries/System.Diagnostics.EventLog/tests/Helpers.cs

```diff
@@ -13,8 +14,9 @@ namespace System.Diagnostics.Tests
 {
     internal class Helpers
     {
-        public static bool NotElevatedAndSupportsEventLogs { get => !AdminHelpers.IsProcessElevated() && SupportsEventLogs; }
-        public static bool IsElevatedAndSupportsEventLogs { get => AdminHelpers.IsProcessElevated() && SupportsEventLogs; }
+        public static bool HasAssemblyFilesIsElevatedAndSupportsEventLogs => PlatformDetection.HasAssemblyFiles && IsElevatedAndSupportsEventLogs;
```

Comment:
This one's only used by the test being fixed in this PR, so I can safely switch it to check if RemoteExecutor is supported.

```suggestion
        public static bool HasAssemblyFilesIsElevatedAndSupportsEventLogs => PlatformDetection.HasAssemblyFiles && RemoteExecutor.IsSupported && SupportsEventLogs;
```